src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -31,100 +32,150 @@ function ProductGallery({ title }: Props) {
       return
     }
 
-    // notebook breakpoint = 1280px (See breakpoints on styles/global.scss)
-    setIsMobile(screenWidth < 1280)
+    const notebookBreakpoint =
+      getComputedStyle(document.documentElement).getPropertyValue(
+        '--breakpoint-notebook'
+      ) || '1280'
+
+    setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
   }, [screenWidth])
 
-  if (!data) {
-    return <div className="temp-data-loading">loading...</div>
+  const orderedFacets = useOrderedFacets(data)
+
+  if (!orderedFacets.length) {
+    return <div className="product-listing__data-loading">Loading…</div>
   }
 
   return (
-    <>
-      {/* Filters */}
-      <div>
-        <Filter
-          isOpen={isFilterOpen}
-          facets={data.search.facets}
-          onDismiss={() => setIsFilterOpen(false)}
-        />
-        {isMobile && (
-          <Button
-            variant="tertiary"
-            data-testid="open-filter-button"
-            onClick={() => setIsFilterOpen(!isFilterOpen)}
-          >
-            <UIIcon component={<FadersHorizontalIcon size={18} />} />
-            Filters
-          </Button>
+    <div className="product-listing / grid-content-full">
+      <div className="product-listing__content-grid / grid-content">
+        <div className="product-listing__filters">
```

Comment:
I notice the title (`position: sticky`) overlapping the `Filter` component on the top left corner:
![ickQZQNahF](https://user-images.githubusercontent.com/11325562/151265394-9ff70a58-5a66-4a67-8309-b16bea9a805d.gif)

Reply:
hmm @filipewl can't we put `product-listing__filters-bar` part inside `product-listing__filters`, and then apply `position: sticky` there? 👀

Reply:
@hellofanny we had that before but the "Filters" heading was misaligned with the right of the row ("X Results" and "Sort By"). I and @renatamottam reworked the grid to keep heading the filters separated, which is this current version.

![CleanShot 2022-01-27 at 16 26 01](https://user-images.githubusercontent.com/381395/151429555-df28f2b0-047d-4346-80c0-f9a974cd3079.png)

We didn't notice this was happening and as there's no other way to fix protect the heading, we'll have to revert and have them both wrapped in the same element.

Reply:
Fixed it in [this commit](https://github.com/vtex-sites/base.store/commit/786d00dcb82a804fb732f4976cd2b450e41f5bf6) for another PR.